sdk/python/tests/unit/permissions/conftest.py

```diff
@@ -49,6 +49,7 @@ def users() -> list[User]:
     users.append(User("w", ["writer"]))
     users.append(User("rw", ["reader", "writer"]))
     users.append(User("admin", ["reader", "writer", "admin"]))
+    users.append(User("test1234", []))
```

Comment:
I don't think we need this entry, it should mock the users from the IDP server instead. can you try w/o it?